hlx_statics/blocks/header/header.js

```diff
@@ -52,15 +52,16 @@ function globalSignIn() {
 function globalNavLinkItemDropdown(id, name, links) {
   return `
       <button id="nav-dropdown-button_${id}" class="spectrum-Picker spectrum-Picker--sizeM spectrum-Picker--quiet navigation-dropdown" aria-haspopup="listbox">
-        <span class="spectrum-Picker-label">
+        <span class="spectrum-Picker-label nav-label">
           ${name}
         </span>
-        <svg class="spectrum-Icon spectrum-UIIcon-ChevronDown100 spectrum-Picker-menuIcon" focusable="false" aria-hidden="true">
-          <use xlink:href="#spectrum-css-icon-Chevron100" />
-        </svg>
+        <svg aria-hidden="true" role="img" focusable="false" class="spectrum-Icon spectrum-UIIcon-ChevronDown100 spectrum-Picker-menuIcon dropdown-icon">
+          <path d="M4.5 13.25a1.094 1.094 0 01-.773-1.868L8.109 7 3.727 2.618A1.094 1.094 0 015.273 1.07l5.157 5.156a1.094 1.094 0 010 1.546L5.273 12.93a1.091 1.091 0 01-.773.321z" class="spectrum-UIIcon--large"></path>
+          <path d="M3 9.95a.875.875 0 01-.615-1.498L5.88 5 2.385 1.547A.875.875 0 013.615.302L7.74 4.377a.876.876 0 010 1.246L3.615 9.698A.872.872 0 013 9.95z" class="spectrum-UIIcon--medium"></path>
+         </svg>
       </button>
-      <div id="nav-dropdown-popover_${id}" class="spectrum-Popover spectrum-Popover--bottom spectrum-Picker-popover spectrum-Picker-popover--quiet filter-by-popover nav-dropdown-popover">
-        <ul class="spectrum-Menu" role="menu">
+      <div id="nav-dropdown-popover_${id}" class="nav-dropdown-popover">
```

Comment:
I think we still need `spectrum-Popover` classes. I'm testing the menu drop down effect when not in mobile mode and the dropdown doesn't appear.